### PR TITLE
Remove warning ‘GTimeVal’ is deprecated

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: pluma/pluma-gio-document-saver.c [hunk 1]
```diff
@@ -739,7 +748,7 @@ save_remote_file_real (PlumaGioDocumentSaver *gvsaver)
 
 static void
 pluma_gio_document_saver_save (PlumaDocumentSaver *saver,
-			       GTimeVal           *old_mtime)
+			       gint64             *old_mtime)
```
Comment (raveit65) [inline pluma/pluma-gio-document-saver.c:751]: Indent!

Comment (rbuj) [reply pluma/pluma-gio-document-saver.c]: done

**Comment by raveit65** [comment]
Where does all the left over `GTimeVal`deprecation warnings in build log comes from?
From GtkSourceView itself?

**Comment by selectiveduplicate** [comment]
The rest of the `GTimeVal` errors I think can be avoided by using `guint64` or `GDateTime` instead, according to the docs.

**Comment by raveit65** [comment]
Ok, this can be done in another PR.

**Pull request merged**